compiler_gym/spaces/reward.py

```diff
@@ -38,6 +38,8 @@ class Reward(Scalar):
 
     def __init__(
         self,
+        # TODO(cummins): Rename `id` to `name` for consistency with the other
```

Comment:
Do you want to implement this TODO in this PR or in a future PR?

Reply:
Good question. I've filed an issue and updated the comment to reference it. #381.